io.asgardeo.tomcat.oidc.sample/src/main/resources/oidc-sample-app.properties

```diff
@@ -17,19 +17,22 @@
 #
 consumerKey=<OAuth Client Key>
 consumerSecret=<OAuth Client Secret>
-skipURIs=/oidc-sample-app/index.html
-indexPage=
-errorPage=/error.jsp
-logoutURL=logout
-callBackURL=http://localhost:8080/oidc-sample-app/oauth2client
 scope=openid,address,email,profile
 #grantType=code
+signatureAlgorithm=RS256
+
+callBackURL=http://localhost:8080/oidc-sample-app/oauth2client
+postLogoutRedirectURI=http://localhost:8080/oidc-sample-app/index.html
+trustedAudience=http://localhost:8080/sample-app
+skipURIs=/oidc-sample-app/index.html
+indexPage=index.html
+logoutURL=logout
+errorPage=/error.jsp
+
+#Asgardeo endpoints
 authorizeEndpoint=https://localhost:9443/oauth2/authorize
```

Comment:
Let's take issuer to the start of endpoints or even outside of it.

Reply:
Or we can name it as #Asgardeo related configs

Reply:
can we rename it as Asgardeo related configs ->`IdP related configs`